lib/Sema/CSGen.cpp

```diff
@@ -124,11 +124,7 @@ namespace {
       
       return Action::Continue(expr);
     }
-    
-    PostWalkResult<Expr *> walkToExprPost(Expr *expr) override {
-      return Action::Continue(expr);
-    }
-    
+
```

Comment:
All of the SkipChildren here should be `SkipChildrenAndPostWalk`, I don't think we care about post walk for any of the ignored things...